packages/gatsby-theme-api-docs/src/components/type/properties/rows/description.js

```diff
@@ -38,30 +37,46 @@ const DescriptionText = styled.span`
 `;
 
 const Description = ({ property, discriminatorValue }) => {
+  const isConstantLike =
+    property.enumeration && property.enumeration.length === 1;
   const additionalInfos = extractAdditionalInfo(property);
   const renderEnums = property.enumeration && !discriminatorValue;
   return (
     <SpacingsStack scale="s">
-      <DescriptionText>
-        {markdownFragmentToReact(property.description)}
-      </DescriptionText>
-      <SpacingsInline>
-        {renderEnums ? (
-          <Info>
-            {valuesStr}: {property.enumeration.join(', ')}
-          </Info>
-        ) : null}
-        {Object.entries(additionalInfos).map(([info, value], index) => {
-          return (
-            !(typeof value === 'boolean' && !value) && (
-              <Info key={index}>
-                {info}
-                <InfoValue>{value}</InfoValue>
+      {isConstantLike ? (
+        <DescriptionText>
+          <div>
+            <Markdown.InlineCode>{property.enumeration[0]}</Markdown.InlineCode>
+          </div>
+          <div>
+            Only &quot;{property.enumeration[0]}&quot; value must be assigned to{' '}
+            {property.name}
+          </div>
+        </DescriptionText>
+      ) : (
+        <>
+          <DescriptionText>
+            {markdownFragmentToReact(property.description)}
+          </DescriptionText>
+          <SpacingsInline>
+            {renderEnums ? (
+              <Info>
+                Value must be one of: {property.enumeration.join(', ')}
```

Comment:
- the text is still cumbersome.  For inspiration: Stripe has a mix of automated and hand-authored wordings here: https://stripe.com/docs/api/accounts/object    "Possible values: X, Y, Z",  "Values may be X, Y or Z", "Can be X, Y or Z".   For a visual component like here I like "Possible values: " best (better than the "Allowed values: " I originally proposed). 
- Please try to render the values depending on the property type (string vs. number) and also apply that logic in other places like "default".